cloudformation/lambda/auto_off_beanstalk/auto_off_beanstalk.yaml

```diff
@@ -22,18 +22,7 @@ Resources:
           Path: /
           ManagedPolicyArns:
           - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
-          Policies:
-                - PolicyName: PublishAndExecute
-                  PolicyDocument:
-                    Version: 2012-10-17
-                    Statement:
-                    - Effect: Allow
-                      Action: [
-                        'elasticbeanstalk:TerminateEnvironment',
```

Comment:
Not sure why you need this. It wont make @lwgordonimos happy!

Reply:
Wasn't working otherwise. The "terminate" call was successful, but the actual termination was failing because it didn't have the needed permissions to delete the underlying components.

Reply:
Fixed now so that the functions can delete to their hearts' content, which we need, but can't create anything